Resources/Textures/Structures/Furniture/toilet.rsi/meta.json

```diff
@@ -5,23 +5,239 @@
         "y": 32
     },
     "license": "CC-BY-SA-3.0",
-    "copyright": "Taken from cev-eris at commit https://github.com/discordia-space/CEV-Eris/commit/2cb66bae0e253e13d37f8939e0983bb94fee243e",
+    "copyright": "Taken from https://github.com/goonstation/goonstation/blob/master/icons/obj/objects.dmi and resprited by brainfood1183",
     "states": [
```

Comment:
license must be changed to `CC-BY-NC-SA-3.0` if its goon sprites

Reply:
also blob master :trollface: